api/src/main/java/org/apache/iceberg/actions/ExpireSnapshots.java

```diff
@@ -101,6 +101,16 @@ interface Result {
      */
     long deletedDataFilesCount();
 
+    /**
+     * Returns the number of deleted equality delete files.
+     */
+    long deletedEqualityDeleteFilesCount();
+
+    /**
+     * Returns the number of deleted positional delete files.
+     */
+    long deletedPositionalDeleteFilesCount();
```

Comment:
I think we use `positionDeleteFiles`, not `positionalDeleteFiles`. Better be consistent.